agent/check_handler.go

```diff
@@ -19,9 +20,12 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const allowListOnDenyStatus = "allow_list_on_deny_status"
+const allowListOnDenyOutput = "Check command denied by the Agent Allow list"
```

Comment:
Any reason why `Agent Allow` are capitalized but not list? Also, error strings should usually not be capitalized (https://github.com/golang/go/wiki/CodeReviewComments#error-strings).

Reply:
I literally just copied and pasted the string from the spec 😆I've fixed this in the check and hook error messages!